test/cypress/e2e/registration.spec.js

```diff
@@ -17,7 +17,7 @@ describe('Registration', () => {
    * Create an account
    * @expect Account is created
    */
-  it('Create an account', function () {
+  xit('Create an account', function () {
```

Comment:
The design changed and there is no more account passphrase rendered as a string
`    cy.get(ss.copyPassphrase).invoke('text').as('passphrase');`

Reply:
@massao Suggestion
` /**
   * Create an account
   * @expect Account is created
   */
  it('Create an account', function () {
    cy.visit(urls.register);
    cy.get(ss.chooseAvatar).first().click();
    cy.get(ss.getPassphraseButton).click();
    this.passphrase = [];
    cy.get(ss.copyPassphrase).each(($el) => {
      this.passphrase = [...this.passphrase, $el.val()];
    });
    cy.get(ss.itsSafeBtn).click();
    cy.get(ss.passphraseWordConfirm).each(($el) => {
      if (this.passphrase.includes($el[0].textContent)) cy.wrap($el).click();
    });
    cy.get(ss.passphraseConfirmButton).click();
    cy.get(ss.loginBtn).click();
  });`

Reply:
@Efefefef Since we don't have the entire passphrase in only one element anymore, what do you think of the solution above? Or maybe you have a better solution to this case.

Reply:
@michaeltomasik I think that's the legitimate way